pandera/model.py

```diff
@@ -98,6 +98,9 @@ class BaseConfig:  # pylint:disable=R0903
     #: validate MultiIndex in order
     multiindex_ordered: bool = True
 
+    #: make sure dataframe column names are unique
+    allow_duplicate_column_names: bool = True
```

Comment:
I think for naming consistency pandera's options lean more towards the affirmative with respect to whatever the validation constraint is... i.e. in this case `unique_column_names: bool = False` instead of `allow_duplicate_column_names`. Let me know what you think.

Reply:
That seems fine to me, it's more concise